File: src/main/java/pl/polsl/projectmanagementsystem/model/Presence.java
package pl.polsl.projectmanagementsystem.model;

import lombok.*;

import javax.persistence.*;
import java.time.OffsetDateTime;
import java.util.Date;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Presence {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Boolean wasPresent = false;

    @ManyToOne
    private Meeting meeting;

    @ManyToOne
    private StudentGroup studentGroup;




}

File: src/app/model/presence.ts
import { Meeting } from './meeting';
import { StudentGroup } from './student-group';

export class Presence {
  id?: number;

  wasPresent: boolean;
  meetingDate: Date;

  meetingId?: number;

  studentGroupId?: number;

  constructor() {
    this.wasPresent = false;
    this.meetingDate = new Date();
  }
}
